config/autoload/.gitignore

```diff
@@ -1,2 +1,3 @@
 local.php
 *.local.php
+event_store.local.php.copy
```

Comment:
This is for integration tests only. The app needs a config in order to run within integration tests. If a config exists already, it will be moved to `event_store.local.php.copy` and restored after the tests. If the tests break and die, the file will be left. I don't want it accidentally committed to repo then.